### DISCUSSION TITLE
SSL: SSLV3_ALERT_HANDSHAKE_FAILURE AIRFLOW

### DISCUSSION THREAD

**Discussion by ellouminisrine** [apache/airflow#27295] [Q&A]
Apache Airflow version:
2.0.2

While accesing an API (sendinblue) using AIRFLOW 2.0.2 and on GCP composer 2.0.2, I had the following incident:The connection between the AIRFLOW agent and the sendinblue server is lost due to an SSL handshake error (the error happened while accessing the get request in a for loop, some hours later after the start of the DAG).

urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/contacts/[id_Client]/campaignStats (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1131)')))

**Comment by potiuk** [comment]
Nothinin Airlfow'y - it's our SSL problem and needs to be further investigated by you - can be related to bad certificates, old TLS used on your machine and likely many other things.
